ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -18,6 +18,37 @@ presubmits:
           requests:
             cpu: 10m
     trigger: ((?m)^/test promrules,?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/build-farm-consistency
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-release-master-build-farm-consistency
+    rerun_command: /test build-farm-consistency
+    spec:
+      containers:
+      - args:
+        - -c
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+          pwd
+          ls -al
+          echo "${GOPATH}/src/github.com/openshift/release/"
```

Comment:
This is your working directory, why do you need to `-C`?

Reply:
Removed.
Debugging when i have `skip_clone: true`